ui/Rhino/AGS/dev/AGS_form_displaysettings_cmd.py

```diff
@@ -69,7 +69,7 @@ def RunCommand(is_interactive):
             elif show == "False":
                 form.artist.settings['show.forces'] = False
 
-        elif option == "ScaleForces":
+        elif option == "PipeScale":
```

Comment:
not a big fan of the word "PipeScale"

Reply:
@ricardoavelino any nice idea? I am running out of inspiration...

Reply:
The problem with the current name is that `ScaleForces` and `Force Scale`, which are things completely different are renamed too similarly and this created confusion for users. The future menu with tabs will end this confusion. For now we just need two names that are not so close to each other...

Reply:
@tomvanmele  any fancy name?

Reply:
I am ok with `PipeScale`, but here are some other ideas `ThicknessScale`, `AxialForceScale`, `AxialScale`, `MaterialScale`...

Reply:
`AxialForceScale`?